Broken on latest nightly

Compiletest does not build on today's nightly:

```rust
   Compiling compiletest_rs v0.2.4
error[E0063]: missing fields `filter_exact`, `list` in initializer of `test::TestOpts`
   --> /Users/sbenitez/.cargo/registry/src/github.com-1ecc6299db9ec823/compiletest_rs-0.2.4/src/compiletest.rs:129:5
    |
129 |     test::TestOpts {
    |     ^^^^^^^^^^^^^^ missing `filter_exact`, `list`

error: aborting due to previous error

error: Could not compile `compiletest_rs`.
```

--- a/src/common.rs
+++ b/src/common.rs
@@ -127,6 +127,9 @@ pub struct Config {
     // Only run tests that match this filter
     pub filter: Option<String>,
 
+    // Exactly match the filter, rather than a substring
+    pub filter_exact: bool,
+
     // Write out a parseable log of tests that were run
     pub logfile: Option<PathBuf>,
 

--- a/src/compiletest.rs
+++ b/src/compiletest.rs
@@ -61,6 +61,7 @@ pub fn default_config() -> Config {
         mode: Mode::RunPass,
         run_ignored: false,
         filter: None,
+        filter_exact: false,
         logfile: None,
         runtool: None,
         host_rustcflags: None,
@@ -128,6 +129,7 @@ pub fn run_tests(config: &Config) {
 pub fn test_opts(config: &Config) -> test::TestOpts {
     test::TestOpts {
         filter: config.filter.clone(),
+        filter_exact: config.filter_exact,
         run_ignored: config.run_ignored,
         quiet: config.quiet,
         logfile: config.logfile.clone(),
@@ -140,6 +142,7 @@ pub fn test_opts(config: &Config) -> test::TestOpts {
         color: test::AutoColor,
         test_threads: None,
         skip: vec![],
+        list: false,
     }
 }
 

